Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.yml

```diff
@@ -11,61 +11,60 @@ configuration:
   name: email
   required: true
   type: 0
-  defaultvalue:
-  additionalinfo:
+  defaultvalue: null
```

Comment:
nulls should probably be reverted. @anas-yousef is your script making this change?

Reply:
It is done from the script